dpuagent/dpuagent/yuccadriver.py

```
@@ -0,0 +1,208 @@
+
+    def create_rbd_bdev(self, pool_name, image_name, cluster_name):
+        cmd_str = "{script} bdev_rbd_create {pool} {image} 512 --name {name} -c {cluster}".format(
+            script=self.RPC_SCRIPT,
+            pool=pool_name,
+            image=image_name,
+            name=image_name,
+            cluster=cluster_name
+        )
+
+        logger.debug("Creating bdev: %s", cmd_str)
+        ret, stdout, stderr = bash.bash_roe(cmd_str)
```

Ruff (0.15.2)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)